commonmark-ext-gfm-tables/src/main/java/org/commonmark/ext/gfm/tables/HtmlTablesExtension.java

```diff
@@ -20,13 +20,13 @@
  * The parsed tables are turned into {@link TableBlock} blocks.
  * </p>
  */
-public class TablesExtension implements Parser.ParserExtension, HtmlRenderer.HtmlRendererExtension {
+public class HtmlTablesExtension implements Parser.ParserExtension, HtmlRenderer.HtmlRendererExtension {
```

Comment:
It should not be necessary to split `TablesExtension` into two classes, just keep it as is and implement `TextContentRendererExtension` here. See `StrikethroughExtension` for an example.

That also means we won't break existing users of `TablesExtension`.